File: java/BEE1013.java
import java.util.Scanner;

public class BEE1013 {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int a, b, c, maior;

        // Entrada
        a = teclado.nextInt();
        b = teclado.nextInt();
        c = teclado.nextInt();

        // Processamento
        maior = (a + b + Math.abs(a - b)) / 2;
        maior = (maior + c + Math.abs(maior - c)) / 2;

        // Saída
        System.out.println(maior + " eh o maior");

        teclado.close();
    }
}

File: python/BEE1013.py
def maior(a: int, b: int) -> int:
    return int((a + b + abs(a - b)) / 2)


entrada = input().split(" ")

a = int(entrada[0])
b = int(entrada[1])
c = int(entrada[2])

print(f"{maior(maior(a, b), c)} eh o maior")
